public/uv/lib/AVComponent.js

```diff
@@ -124,14 +124,16 @@ var IIIFComponents;
             //private _lastCanvasWidth: number | undefined;
             _this._lowPriorityFrequency = 250;
             _this._mediaSyncMarginSecs = 1;
-            _this._rangeSpanPadding = 0.0;
+            _this._rangeSpanPadding = 0.25;

```

Comment:
This removes all our local patches to the AVComponent, which means it'll start skipping content again.

Reply:
I'm sorry, I'll pull this into the `pulibrary` fork and see about preserving our customizations from there.

Reply:
I could look into getting them upstream too, but it'd probably take a while.